docs/dictionary-data.md

```diff
@@ -1,76 +1,120 @@
 
-        the following info is also needed, but happily comes from the paradigm
-        layout files right now:
-
-        paradigm layout list used for generating wordforms
-            note: may be empty because indeclinable,
-            or because we just haven’t created a paradigm and/or FST that can
-            handle it
-
-Note that we will sometimes have inflectional categories for `?` = ‘unknown’
- or `V?` = ‘verb with unknown inflectional category’
-
-
-dictionary level:
-
-    for each dictionary entry: 
-    - head (could be word, phrase, or morpheme)
-    - inflectional category
-      IPH indeclinable phrase / IPM indeclinable morpheme
-    - FST lemma, if inflectional class is declinable
-      To be clear: if the head is `kinîminâwâw` with analysis
-      `nîmiw+V+AI+Ind+2Pl`, then the FST lemma is `nîmiw`
-    - list of definitions
-
-    - linguistic stem to display on popup
-
-      (always in Arok’s source except for moprhemes; might be list for phrase)
-      Arok’s \stm fields may not have preverbs or reduplication
-
-      relation to FST is, Arok’s stems are used to create the FST lemmas
-      in the lexc
-
-      lexc source code has `<FST lemma>:<enriched FST stem, without trailing -, and maybe with alternate symbols like n2 or n3>`
-
-      e.g., `acitakotêw:acitakot3 VTAt ;`
-
-    https://github.com/giellalt/lang-crk/blob/main/src/fst/stems/verb_stems.lexc#L111-L141
-
-
-
-    - homograph key
-      note that homographs may be in the same inflectional category, may not be
-
-    - static paradigm, only allowed if specific word class is indeclinable
-      e.g., ‘niya’ has paradigm ‘personal pronoun’
-
+  - For each general word class:
+      - The general word class name, e.g., `V`
+      - A prose description
+
+Note that it may be useful to have inflectional categories and general and
+specific word classes for ‘unknown’; both generically, and for more refined
+concepts such as ‘verb with unknown inflectional category.’ This helps to
+keep separate the two distinct concepts of (1) word classes whose members
+are not declinable because that’s how the language works, and (2) word
+classes that should be declinable but where the required linguistic data
+together with a compatible computational model implementation is not
+presently at hand.
+
+## Dictionary entry level
+
+For each dictionary entry, we will need the following pieces of data:
+
+  - The entry head. In most dictionaries this is usually a single word, but
+    could also be a phrase or morpheme.
+
+  - The inflectional category. Plains Cree can use inflectional categories
+    such as `IPH` “indeclinable phrase” and `IPM` “indeclinable morpheme”
+    for non-word heads.
+
+  - The FST lemma. This is specified iff the inflectional class is
+    declinable. To be clear: the FST lemma is the thing that gets plugged
+    into a paradigm layout template to generate associated wordforms. For
+    example, if the head is `kinîminâwâw` with FST analysis
```

Comment:
If we want to present an example of a head that is not a lemma, I'd use: _nîminâniwan_ <- `nimîw+V+AI+Ind+X`